docs/navigation.md

```diff
@@ -0,0 +1,128 @@
+
+To the right side of the title bar there are three indicators of the Diagnostics window: 
+
+- Valid/Invalid - It is green if the syntax is green and red when invalid.
+- Autosave - It shows whether Autosave is on or off.
+- Type - Shows the language in which the API is written: `JSON` or `YAML`.
+
+Below the title bar there is a control panel. You can the buttons to:
+
+- Hide and show erros
+- Hide and show warnings
+- Hide and show information messages
+- Hide and show hints
+
+There is also search bar to search that aids in finding issues by their names. You will see Settings icon to the right. It will take you to the governance of Diagnostic's settings window.
+
+# HTML Preview
+
+This section displays the preview of everything you have written in the code editor. It uses different fonts, colors, and styles for readability. Each section has a different card. 
+
+It starts with displaying the title, version, description of the API. It renders the formatting too. You can expand each element too see the `key: value` pair of each element, type, payload, and message.
+
+
+# Blocks visualizer
+
+Blocks visualizer is the USP of this product. It gives you a quick overview of your API. Unlike HTML preview, Blocks visualizer gives a diagrammatic visual of your API. 
+
+Title card is at the top-left corner of the Block visualizer window. Followed by the Operations card.  You can visualize channels, published messages, payload, type,  patterns, and relationship between the components.
+
+The Blocks visualizer has two set of controls:
+
+- In the top right corner of the window you will find the play and reset button. Play to view live relationship between the components and its flow. Reset to get back to the default view.
+
+- To bottom left of the window there are four controls: zoom in, zoom out, full-screen, toogle interactivity. You zoom the cards in and out or view them in full-screen mode. To prevent the cards from shifting use the toggle interactivity control.
+
+
+# New File
+
+To create a new file you can pick one of the available templates.
+
+## Templates
+
+There are 5 available to quick start Studio. 
+
+- Simple Example - A basic example of a service that is in charge of processing user signups. Great place to start learning AsyncAPI.
+
+- Apache Kafka - A framework implementation of a software bus using stream-processing. Open Source developed by the Apache Software Foundation.
+
+- WebSocket - A framework implementation of a software bus using stream-processing. Open Source developed by the Apache Software Foundation.
+
+- MQTT - A protocol for fetching resources. It is the foundation of any data exchange on the Web and it is a client-server protocol.
+
+- HTTP - A protocol for fetching resources. It is the foundation of any data exchange on the Web and it is a client-server protocol.
+
+## Real World Examples
+
+Studio provides two real world examples templates to help you get started.
```

Comment:
This sentence can be rephrased or grammatically simplified. typo error : `real-world example templates`